sdks/java/io/hadoop-input-format/src/test/java/org/apache/beam/sdk/io/hadoop/inputformat/unit/tests/inputs/TestEmployeeDataSet.java

```diff
@@ -0,0 +1,65 @@
+package org.apache.beam.sdk.io.hadoop.inputformat.unit.tests.inputs;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.beam.sdk.values.KV;
+import org.apache.hadoop.io.Text;
+
+import com.google.common.base.Function;
+import com.google.common.collect.Lists;
+
+public class TestEmployeeDataSet {
+  /**
+   * Test Utils used in {@link NewObjectsEmployeeInputFormat} and {@link ReuseObjectsEmployeeInputFormat}
+   * for computing splits.
+   */
+  public static final long NUMBER_OF_RECORDS_IN_EACH_SPLIT = 5L;
+  public static final long NUMBER_OF_SPLITS = 3L;
+
+  private static final List<KV<String, String>> data = new ArrayList<KV<String, String>>();
+
+  /**
+   * Returns List of employee details. Employee details are available in the form of {@link KV} in
+   * which, key indicates employee id and value indicates employee details such as name and address
+   * separated by '_'. This is data input to {@link NewObjectsEmployeeInputFormat} and
+   * {@link ReuseObjectsEmployeeInputFormat}.
+   */
+  public static List<KV<String, String>> populateEmployeeData() {
```

Comment:
is there a reason this isn't in a static constructor for the class? does it need to be called publicly?